addons/project/views/project_project_views.xml

```diff
@@ -222,6 +222,7 @@
             <field name="arch" type="xml">
                 <tree decoration-muted="active == False" string="Projects" multi_edit="1" sample="1" default_order="is_favorite desc, sequence, name, id" js_class="project_project_list">
                     <field name="sequence" column_invisible="True"/>
+                    <field name="name" invisible="1"/>
```

Comment:
Hello @MerlinGuillaume 

Can you do another PR to fix that? 
```suggestion
                    <field name="name" column_invisible="1"/>
```

Reply:
Sure 🙂

Reply:
Nevermind, just saw your PR